shell/common/animator.cc

```diff
@@ -65,7 +70,17 @@ void Animator::BeginFrame(ftl::TimePoint frame_time) {
   // TODO(abarth): We should use |frame_time| instead, but the frame time we get
   // on Android appears to be unstable.
   last_begin_frame_time_ = ftl::TimePoint::Now();
+  // We use less than 1/60 second to account for delays between the vsync
+  // notification and this calculation.
+  dart_frame_deadline_ = Dart_TimelineGetMicros() + 15000;
```

Comment:
We shouldn't hard-code this number. Ideally we'd derive it from the time between when the last frame ended and the next one started, or some such.

Reply:
This time is now derived from the actual refresh rate.